rules/rules.libsonnet

```diff
@@ -56,6 +56,51 @@
           },
         ],
       },
+      {
+        name: 'noobaa-odf.rules',
+        rules: [
+          {
+            record: 'odf_system_health_status',
```

Comment:
Don't we need to change the values of 1 and 2 and their mappings? Or are they already aligned?

Reply:
https://github.com/noobaa/noobaa-core/blob/48d27e51a5e61ea1a5d7f77aa6c3063f12723e67/src/server/system_services/stats_aggregator.js#L977-L980

so we need to aligned in core...

Reply:
that would change the ui ... cant we change in rule only?

Reply:
We will expose new metrics that return what we need in core. I will put the PR here for refrance.

Reply:
https://github.com/noobaa/noobaa-core/pull/6718

Reply:
Now that is in We are calling `NooBaa_odf_health_status`